gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -609,7 +613,8 @@ MM_VerboseHandlerOutput::handleExclusiveStart(J9HookInterface** hook, uintptr_t
 	}	
 	writer->formatAndOutput(env, 0, "<exclusive-start %s intervalms=\"%llu.%03.3llu\">", tagTemplate, deltaTime / 1000, deltaTime % 1000);
 	writer->formatAndOutput(env, 1, "<response-info timems=\"%llu.%03.3llu\" idlems=\"%llu.%03.3llu\" threads=\"%zu\" lastid=\"%p\" lastname=\"%s\" />",
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

> - Start the parameter list on a new line. Insert a newline immediately after the opening bracket.
> - Indent the parameter list by at least 1 tab. Where reasonable, the parameter list should be aligned with the opening bracket.
> - Use the same indent for each line of parameters.